src/seedu/addressbook/data/tag/Tagging.java

```diff
@@ -0,0 +1,72 @@
+/**
+ * 
+ */
+package seedu.addressbook.data.tag;
+
+import seedu.addressbook.data.person.Person;
+
+/**
+ * @author Peixuan
+ * Represents tagging for a Person. 
+ */
+public class Tagging {
+	
+	private static final String ADD_SYMBOL = "+";
+	private static final String DELETE_SYMBOL = "-";
+	private static final String FORMAT_TAG_PREFIX = "[";
+	private static final String FORMAT_TAG_SUFFIX = "]";
+	private static final String SPACE_DELIMITER = " ";
+	
+	private Person person;
+	private Tag tag;
+	private Action action;
+	
+	public static enum Action {
+		ADD, DELETE
+	}
+	
+	public Tagging(Person person, Tag tag, Action action) {
+		this.person = person;
+		this.tag = tag;
+		this.action = action;
+	}
+	
+	public Person getPerson() {
+		return this.person;
+	}
+	
+	public Tag getTag() {
+		return this.tag;
+	}
+
+	private String getFormattedTag() {
+		return FORMAT_TAG_PREFIX + this.tag + FORMAT_TAG_SUFFIX;
+	}
+	
+	public String getTagSymbol() {
+		if (this.action == Action.ADD) {
+			return ADD_SYMBOL;
+		} else {
+			return DELETE_SYMBOL;
```

Comment:
Please fix the indentation and change the tab to spaces in Eclipse, see [here](http://stackoverflow.com/questions/407929/how-do-i-change-eclipse-to-use-spaces-instead-of-tabs)